__tests__/Fuzzy.test.ts

```diff
@@ -0,0 +1,45 @@
+// ./__tests__/Fuzzy.test.ts
+//
+// Unittests for the Fuzzy class.
+
+// Local Fuzzy class import.
+import Fuzzy from '../src/Fuzzy';
+
+// Fuzzy class test suite.
+describe('Fuzzy', () => {
```

Comment:
```suggestion
describe('Test Fuzzy', () => {
```
Let's add a `Test` in each describe block just to make it more consistent

Reply:
- [x] Renamed in e3264f8.